modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc

```diff
@@ -416,7 +416,6 @@ the filenames or folder structures. These files are never encrypted:
 * Existing files in the trash bin & Versions. Only new and changed files
 after encryption is enabled are encrypted.
 * Existing files in Versions
-* Image thumbnails from the Gallery app
```

Comment:
Image thumbnails are independent of the Gallery app. I think I wrote this in another PR already: Please just remove "from the Gallery app".

Question: Why is this same text written in three different files?

Reply:
If I understand your intent correctly, I'll move the text to an includeable file.